nav2_msgs/CMakeLists.txt

```diff
@@ -9,36 +9,40 @@ find_package(geometry_msgs REQUIRED)
 find_package(rosidl_default_generators REQUIRED)
 find_package(std_msgs REQUIRED)
 find_package(action_msgs REQUIRED)
+find_package(sensor_msgs REQUIRED)
 
 nav2_package()
 
 rosidl_generate_interfaces(${PROJECT_NAME}
-  "msg/Costmap.msg"
-  "msg/CostmapMetaData.msg"
-  "msg/CostmapFilterInfo.msg"
-  "msg/SpeedLimit.msg"
-  "msg/VoxelGrid.msg"
-  "msg/BehaviorTreeStatusChange.msg"
-  "msg/BehaviorTreeLog.msg"
-  "msg/Particle.msg"
-  "msg/ParticleCloud.msg"
-  "srv/GetCostmap.srv"
-  "srv/ClearCostmapExceptRegion.srv"
-  "srv/ClearCostmapAroundRobot.srv"
-  "srv/ClearEntireCostmap.srv"
-  "srv/ManageLifecycleNodes.srv"
-  "srv/LoadMap.srv"
-  "srv/SaveMap.srv"
-  "action/BackUp.action"
-  "action/ComputePathToPose.action"
-  "action/FollowPath.action"
-  "action/NavigateToPose.action"
-  "action/Wait.action"
-  "action/Spin.action"
-  "action/DummyRecovery.action"
-  "action/FollowWaypoints.action"
-  DEPENDENCIES builtin_interfaces geometry_msgs std_msgs action_msgs nav_msgs
-)
+    "msg/Costmap.msg"
```

Comment:
In `CMakeLists.txt` by convention it is used tabulation by 2-spaces, not 4. Please restore it back.